colors/seoul256.vim

```diff
@@ -308,7 +392,7 @@ call s:hi('GitGutterChange', [68, 68], [s:dark_bg + 1, s:light_bg - 2])
 call s:hi('GitGutterDelete', [161, 161], [s:dark_bg + 1, s:light_bg -2 ])
 call s:hi('GitGutterChangeDelete', [168, 168], [s:dark_bg + 1, s:light_bg - 2])
 
-" http://vim.wikia.com/wiki/Highlight_unwanted_spaces     
+" http://vim.wikia.com/wiki/Highlight_unwanted_spaces
```

Comment:
Absoulutely. I see the intent. I screwed up by adding the whole file to the commit (and had an autocommand that strips the whitespace on write).